plugin/core/sessions.py

```diff
@@ -593,6 +593,17 @@ def on_pre_send_notification_async(self, notification: Notification) -> None:
         """
         pass
 
+    def on_register_capability_async(self, registration_id: str, capability_path: str, options: Dict[str, Any]) -> None:
+        pass
+
+    def on_unregister_capability_async(
+        self, registration_id: str, capability_path: str, options: Dict[str, Any]
+    ) -> None:
+        pass
+
+    def on_session_end_async(self) -> None:
+        pass
```

Comment:
Should add documentation if we decide to go forward with it.

Reply:
Yeah. Although I'm not sure if `on_session_end_async` is really needed.